src/components/Image/Image.js

```diff
@@ -0,0 +1,143 @@
+import React, { memo, forwardRef, useRef, useEffect, useState, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import checkProps from '@jam3/react-check-extra-props';
+import { useIntersectionObserver } from '@jam3/react-hooks';
+
+import './Image.css';
```

Comment:
```suggestion
import './ImageSequence.scss';
```
should be supported now